installation.html

```diff
@@ -147,12 +148,13 @@ <h3>Third-party Packages to be Installed</h3>
            <div id='node_js'><h4>node.js</h4>
              <ul><li><a href='http://nodejs.org'>http://nodejs.org</a></li>
                <li>we are using
-                 <ul><li>node-v0.10.24-linux-x64</li></ul>
+                 <ul><li>node-v8.11.3-linux-x64</li></ul>
                </li>
                <li>create a softlink <i>node</i> in the <i>def/packages</i>
                    directory that points to your node directory</li>
                <li>create a softlink <i>node_modules</i> in the <i>def/packages</i>
-                   directory that points to your node_modules directory</li>
+                   directory that points to your node_modules directory. The node_modules 
+                   should include prototype v0.0.5</li>

```

Comment:
The change I made here is to make sure the node_module has the correct content in it. It can also be done by adding a package.json file in the same directory with the Gemfile for Rails app.